src/components/Room/Actions/ToggleFullscreen.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) [2020] SUSE Linux
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE.txt file for details.
+ */
+
+import React, { useEffect, useState } from 'react';
+import { Maximize2 as Maximize, Minimize2 as Minimize } from 'react-feather';
+import { classNames } from '../../../utils/common';
+
+function ToggleFullscreen({elementRef, className}) {
+  const [usingFullscreen, setUsingFullscreen] = useState(false);
+
+  const Icon = usingFullscreen ? Minimize : Maximize;
+
+  const toggleFullscreen = () => {
+    if (!document.fullscreenElement) {
+      elementRef.current.requestFullscreen()
+        .then(() => setUsingFullscreen(true))
```

Comment:
Since there is an event listener for the fullscreenchange, this line is no longer necessary.